src/Auth0.AuthenticationApi/IAuthenticationApiClient.cs

```diff
@@ -86,6 +86,22 @@ public interface IAuthenticationApiClient
         /// </remarks>
         Task<AccessTokenResponse> GetTokenAsync(ResourceOwnerTokenRequest request);
 
+        /// <summary>
+        /// Requests an Access Token using the Passwordless flow through email.
+        /// </summary>
+        /// <param name="request"><see cref="PasswordlessEmailTokenRequest"/> containing request details to exchange a one time password received through email.</param>
+        /// <returns><see cref="Task"/> representing the async operation containing 
+        /// a <see cref="AccessTokenResponse" /> with the requested tokens.</returns>
+        Task<AccessTokenResponse> GetTokenAsync(PasswordlessEmailTokenRequest request);
+
+        /// <summary>
+        /// Requests an Access Token using the Passwordless flow through SMS.
+        /// </summary>
+        /// <param name="request"><see cref="PasswordlessSmsTokenRequest"/> containing request details to exchange a one time password received through SMS.</param>
+        /// <returns><see cref="Task"/> representing the async operation containing 
+        /// a <see cref="AccessTokenResponse" /> with the requested tokens.</returns>
+        Task<AccessTokenResponse> GetTokenAsync(PasswordlessSmsTokenRequest request);
```

Comment:
Good question. The way I see this breaking things is for people creating their own AuthenticationClient by implementing the IAuthenticationApiClient interface. But I would argue that defeats the purpose of this SDK. 

We could move these 2 methods in a separate interface to avoid breaking changes and keep interfaces smaller. But that would diverge from the current structure of things.